content/uc-doc/administration/import_export.md

```diff
@@ -20,6 +20,7 @@ Installing the import tool is required on the exporting and importing system.
 To install it on a debian based system uses the following commands on the command line.
 
 ```sh
+# Debian Buster (10)
```

Comment:
and also python version, as you can have debian 10 with python3.5 (migration from debian 9)

Reply:
I never saw a debian Buster without python3.7, even with migration from Stretch
I am not sure it possible to have Buster without python3.7 ....